### PR TITLE
feat: Support for `SingleSelectionFollowsFocus`

### LINKED ISSUE #12308: [WASM] `ListView` doesn't support `SingleSelectionFollowsFocus = false`
### Current behavior

ListView controls don't seem to support SingleSelectionFollowsFocus  = false.  We have two ListViews using XY navigation between the two and as you navigate it selects the items.

### Expected behavior

The ListView controls do not select the items.

### How to reproduce it (as minimally and precisely as possible)

[ItemRecycling.zip](https://github.com/unoplatform/uno/files/11480996/ItemRecycling.zip)

### Workaround

_No response_

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

_No response_

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #12308 

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?

- Property is not supported

## What is the new behavior?

- Property is supported

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by MartinZikmund** [comment]
It seems this only works in case of keyboard navigation, so there is no way to properly test this without keyboard injection.

File: src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs [hunk 1]
```diff
@@ -130,7 +131,8 @@ private bool TryMoveKeyboardFocusAndSelection(int offset, SelectorItem focusedCo
 			}
 
 			// If selection mode is single, moving focus also selects the item
-			if (SelectionMode == ListViewSelectionMode.Single)
+			if (SelectionMode == ListViewSelectionMode.Single &&
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs:134]: Are we able to test this new feature with automation peers?

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs:134]: I think it is not possible, as the only way to get into this portion of code is from `OnKeyDown`, and programmatic focus does not seem to move selection this way even if `SingleSelectionFollowsFocus` on UWP/WinUI

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs:134]: Got it. Let's mark the UI Test as a manual test then.

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs:134]: @jeromelaban Done!

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**